workflows/diagnostics/fv3net/diagnostics/prognostic_run/computed_diagnostics.py

```diff
@@ -247,9 +247,9 @@ def load_diagnostics(rundirs) -> Tuple[Metadata, Diagnostics]:
         for key, ds in diags.items()
     ]
     diagnostics = [convert_index_to_datetime(ds, "time") for ds in diagnostics]
-    diagnostics = [_add_derived_diagnostics(ds) for ds in diagnostics]
     longest_run_ds = _longest_run(diagnostics)
     diagnostics.append(_get_verification_diagnostics(longest_run_ds))
+    diagnostics = [_add_derived_diagnostics(ds) for ds in diagnostics]
```

Comment:
Switched the order here, which allows computing the conditional average of the joint histograms for the verif data.